util/pingpong.c

```diff
@@ -1953,6 +1977,9 @@ void pp_parse_opts(struct ct_pingpong *ct, int op, char *optarg)
 		ct->opts.dst_port = parse_ulong(optarg, UINT16_MAX);
 		break;
 
+	case 'm':
+		ct->tagged = strncasecmp("msg", optarg, 4);
```

Comment:
yes, but it is simpler to maintain when initialization of hints is located at single function